doc: make README(.md) more informative by indicating that a GPG key u… (#32591)
nodejs/node: README.md

### Release Keys

> GPG keys used to sign Node.js releases (you may encounter a release signed
> by a sub-key of a key listed below (which itself isn't listed below)):

Reviewer: ```suggestion
Primary GPG keys for Node.js Releasers (some Releasers sign with subkeys):
```

Author: Consensus on part 1.

* **Shelley Vohr** &lt;shelley.vohr@gmail.com&gt;
`B9E2F5981AA6E0CD28160D9FF13993A75599653C`

> If you encounter a release signed by a key not listed above, please import it
> and check whether it is a sub-key of a primary key listed above. You can also
> import the full set of trusted release keys:

Reviewer: ```suggestion
To import the full set of trusted release keys:
```
